main/index.js

```diff
@@ -1,4 +1,15 @@
 const { app, ipcMain, protocol, shell, clipboard, globalShortcut, BrowserWindow } = require('electron')
+const log = require('electron-log')
+
+log.transports.console.level = process.env.LOG_LEVEL || 'info'
+log.transports.file.level = ['development', 'test'].includes(process.env.NODE_ENV) ? false : 'verbose'
+
+const hasInstanceLock = app.requestSingleInstanceLock()
+
+if (!hasInstanceLock) {
+  log.info('another instance of Frame is running - exiting...')
+  return app.quit()
```

Comment:
my TS gives an error here because return is not being called from inside a function so doesn't really make sense. better is probably just `app.exit(1)` which exits the process immediately without asking, as opposed to `quit()` which attempts to close open windows and fire some events

Reply:
This isn't a TS file in this branch?  I changed it anyway as sounds reasonable